docs/agents/spec-process.md

```diff
@@ -0,0 +1,58 @@
+Every change that affects more than one agent should be initiated via a change to the specification rather than creating a cross-agent issue.
+
+## Advantages
+
+- Ensures we have an up-to-date [spec](https://www.joelonsoftware.com/2000/08/09/the-joel-test-12-steps-to-better-code/) for agents.
+- Easier to keep track of what the current state of the proposal is.
+- Comments to a specific sentence of the spec can be made inline,
+  just like in code reviews.
+- If there are changes or amendments to a spec later on,
+  a PR to an existing spec is much cleaner than having to track down all preceding issues to find out what the current state is.
+- Easier to get a new agent (in-house or community) started.
+
+## Process
+
+1. Agents discussion issue \
+  If there's a lot of uncertainty about what the poposal is,
+  a discussion issue should be created.
+  In doubt, lean towards not creating a discussion issue and start with a draft PR.
+  Discussion issues should not have votes/agent checkboxes.
+  It's not expected that all agents participate in the initial discussions, although everyone is invited to do so.
+1. Draft PR \
+  It's fine if the first draft of the spec change even if details are unclear and need discussion.
+  The PR should include which agents are affected by this change.
+  Specifically, it should always be clear whether the RUM agent is affected by this.
+  This doesn't have to be determined yet when creating the draft PR,
+  but it is required before marking the PR as `Ready for Review`.
+1. Approval by at least one other agent \
+  This makes sure the change makes sense for not just one agent.
+  This doesn't ping all agent teams yet,
```

Comment:
The only unclear part to me about this step is: how do I get someone to approve the draft? The draft PR can be approved, but if we don't ping the team, then is the person who open the PR expected to just somehow find someone from another agent team to review the PR (e.g. on slack)?

Reply:
I think the spec creator should actively reach out to another team. Ideally, they already know another team that has gotten a similar request. Otherwise, they can ask for a design partner on slack or on a weekly. If all of that fails, a lead has to step in.